OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -390,6 +394,23 @@ void AddAsset(ScrollPanelWidget list, string filepath, IReadOnlyPackage package,
 
 		bool LoadAsset(IReadOnlyPackage package, string filename)
 		{
+			if (currentSprites != null)
+			{
+				currentSprites = null;
+				currentFrame = 0;
+			}
+
+			if (currentVoxel != null)
```

Comment:
There is no harm in setting these to null if they are already null, so how about just
```csharp
if (currentSound != null)
	Game.Sound.StopSound(currentSound);

currentSprites = null;
currentFrame = 0;
currentVoxel = null;
currentSound = null;
```
?